### DISCUSSION TITLE
Having a subform / subtable

### DISCUSSION THREAD

**Discussion by hendrikjap** [Budibase/budibase#4473] [Ideas]
Instead of using modal or full screen form to add related data (with relevant parent child handlebar and relationship), having subform / subtable or inline editing will greatly improve budibase.

**Comment by mjashanks** [comment]
Agreed.

You could technically build out a sub-form your self, by using the structure

```
> Data Provider
  > Repeater
    > Container with a ow of details
    > Form
```

You could use a button inside each row to show/hide the form, using and `Update State` action.

...but not possible to to this with a table component at the moment. Agreed, inline would be very useful.
